app/Controllers/configureController.php

```diff
@@ -16,32 +16,33 @@ public function firstAction(): void {
 		}
 	}
 
-	/**
-	 * This action handles the display configuration page.
-	 *
-	 * It displays the display configuration page.
-	 * If this action is reached through a POST request, it stores all new
-	 * configuration values then sends a notification to the user.
-	 *
-	 * The options available on the page are:
-	 *   - language (default: en)
-	 *   - theme (default: Origin)
-	 *   - darkMode (default: no)
-	 *   - content width (default: thin)
-	 *   - display of read action in header
-	 *   - display of favorite action in header
-	 *   - display of date in header
-	 *   - display of open action in header
-	 *   - display of read action in footer
-	 *   - display of favorite action in footer
-	 *   - display of sharing action in footer
-	 *   - display of article tags in footer
-	 *   - display of my Labels in footer
-	 *   - display of date in footer
-	 *   - display of open action in footer
-	 *   - html5 notification timeout (default: 0)
-	 * Default values are false unless specified.
-	 */
+    /**
+     * This action handles the display configuration page.
+     *
+     * It displays the display configuration page.
+     * If this action is reached through a POST request, it stores all new
+     * configuration values then sends a notification to the user.
+     *
+     * The options available on the page are:
+     *   - language (default: en)
+     *   - theme (default: Origin)
+     *   - darkMode (default: no)
+     *   - content width (default: thin)
+     *   - display of read action in header
+     *   - display of favorite action in header
+     *   - display of date in header
+     *   - display of open action in header
+     *   - display of read action in footer
+     *   - display of favorite action in footer
+     *   - display of sharing action in footer
+     *   - display of article tags in footer
+     *   - display of my Labels in footer
+     *   - display of date in footer
+     *   - display of open action in footer
+     *   - html5 notification timeout (default: 0)
+     * Default values are false unless specified.
```

Comment:
Please make sure only relevant lines are changed before you commit, as much as possible